lib/accumulate_distribute/util/gen_order_amounts.js

```diff
@@ -1,10 +1,39 @@
 'use strict'
 
 const _isFinite = require('lodash/isFinite')
-const { prepareAmount } = require('bfx-api-node-util')
 const { Config } = require('bfx-api-node-core')
+const { prepareAmount } = require('bfx-api-node-util')
+const getMinMaxDistortedAmount = require('../../util/get_min_max_distorted_amount')
+const getRandomNumberInRange = require('../../util/get_random_number_in_range')
+
 const { DUST } = Config
 
+const addRemainingToExistingOrderAmounts = (orderAmounts, opts) => {
+  let { cappedOrderAmount: amountToBeDistributed } = opts
+  const { amount, maxDistortedAmount } = opts
+
+  const maxPossibleAmount = maxDistortedAmount * orderAmounts.length
```

Comment:
```
> 1.3 * 3
3.9000000000000004
```

Reply:
fixed all of the following float issues in the acc/dist PR.